src/test/java/org/fundacionjala/sfdc/stepdefinition/CommonSteps.java

```diff
@@ -2,10 +2,12 @@
 
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.core.driver.DriverManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.core.driver.DriverManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)